test/abstractarray.jl

```diff
@@ -355,29 +358,29 @@ function test_vector_indexing{T}(::Type{T}, shape, ::Type{TestAbstractArray})
     @test B[vec(idxs)] == A[vec(idxs)] == vec(idxs)
     @test B[:] == A[:] == collect(1:N)
     @test B[1:end] == A[1:end] == collect(1:N)
-    @test B[:,:] == A[:,:] == reshape(1:N, shape[1], prod(shape[2:end]))
-    @test B[1:end,1:end] == A[1:end,1:end] == reshape(1:N, shape[1], prod(shape[2:end]))
+    # @test B[:,:] == A[:,:] == reshape(1:N, shape[1], prod(shape[2:end]))
+    # @test B[1:end,1:end] == A[1:end,1:end] == reshape(1:N, shape[1], prod(shape[2:end]))
```

Comment:
I initially tried that, but it doesn't quite work. It breaks when `--depwarn=error`.  And the `depwarn` state is different depending upon how the tests are run (`include("test/…")` vs `make -C test …` vs `make test`).  I suppose I can just tack on a comment with an easy-to-grep-for string on all these lines.